.github/workflows/gene_bump_dry_run.yml

```diff
@@ -0,0 +1,55 @@
+name: Gene Bump Dry Run
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - '**/cellxgene_schema_cli/cellxgene_schema/ontology_files/*.csv.gz'
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  dry-run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
```

Comment:
duplicate note: removesuffix, used in the dry run, was introduced in Python 3.9

Reply:
I don't understand this comment